### PR TITLE
ENG-122 Removed css classes from boltcss.phtml file

### PR DESCRIPTION
**Pull request by njfaries**
# Description
Initial commit for removing css classes from boltcss.phtml. Some of these are used by custom integrations for some m2 merchants, so there will need to be work done to get them put into the global css field on the config page until we can make a more permanent fix by moving options elsewhere. 
Fixes: (link Asana Task)

#changelog Removes classes from boltcss.phtml

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: view/frontend/templates/css/boltcss.phtml [hunk 1]
```diff
@@ -36,11 +32,6 @@ if ($block->shouldDisableBoltCheckout()) return;
         border: 1px solid #cccccc;
         width:200px;
     }
```
Comment (danac-gs) [inline view/frontend/templates/css/boltcss.phtml]: @njfaries the above section can be added to Global CSS configuration defaults and removed from here as well.
https://github.com/BoltApp/bolt-magento2/blob/master/etc/config.xml#L47

**Comment by njfaries** [comment]
For some reason I can't get these changes to be reflected on my devbox for testing, but I've pushed them here for review. I believe this should work.

File: view/frontend/templates/css/boltcss.phtml [hunk 2]
```diff
@@ -20,27 +20,6 @@
  *
  * @var $block \Bolt\Boltpay\Block\Js
  */
-if ($block->shouldDisableBoltCheckout()) return;
+if ($block->shouldDisableBoltCheckout()) ;
```
Comment (danac-gs) [inline view/frontend/templates/css/boltcss.phtml]: you need to return / stop here if the condition is matched

File: view/frontend/templates/css/boltcss.phtml [hunk 3]
```diff
@@ -20,27 +20,6 @@
...
 ?>
-<style>
```
Comment (danac-gs) [inline view/frontend/templates/css/boltcss.phtml]: You need the <style> tag to inject CSS into. Just remove the hardcoded CSS, leave the rest as is, that's it.

**Pull request merged**